roles/confluent.ksql/templates/ksql-server_log4j.properties.j2

```diff
@@ -44,13 +44,57 @@ log4j.appender.kafkaAppender.layout.ConversionPattern=[%d] %p %m (%c:%L)%n
 log4j.appender.kafkaAppender.MaxFileSize={{ksql.kafka_appender_log_file_size}}
 log4j.appender.kafkaAppender.MaxBackupIndex={{ksql.kafka_appender_max_log_files}}
 log4j.appender.kafkaAppender.append=true
+{% if ksql_log_streaming_enabled | bool %}
+ksql.logging.processing.topic.auto.create: {{ksql_log_streaming_enabled | lower}}
+ksql.logging.processing.topic.name: {{ksql_service_id}}{{ksql_processing_log}}
 
+# Log Streaming Configuration
 log4j.appender.kafka_appender=org.apache.kafka.log4jappender.KafkaLog4jAppender
 log4j.appender.kafka_appender.layout=io.confluent.common.logging.log4j.StructuredJsonLayout
-log4j.appender.kafka_appender.BrokerList=localhost:9092
-log4j.appender.kafka_appender.Topic=default_ksql_processing_log
+log4j.appender.kafka_appender.BrokerList={% for host in groups['kafka_broker'] %}{% if loop.index > 1%},{% endif %}{{ host }}:{{kafka_broker_listeners[kafka_broker_inter_broker_listener_name]['port']}}{% endfor %}
+
+log4j.appender.kafka_appender.Topic={{ksql_processing_log}}
 log4j.logger.processing=ERROR, kafka_appender
 
+# Security Configuration for Log Streaming
+{% set listener = kafka_broker_listeners[kafka_broker_inter_broker_listener_name] %}
```

Comment:
maybe the listener should only be the interbroker listener in an rbac enabled situation. if rbac is disabled it would make more sense to use ksql listener, wdyt?

I would create a var like:
```
ksql_processing_log_listener: "{{kafka_broker_inter_broker_listener_name if rbac_enabled else ksql_listener}}"
```

Reply:
We enable two listeners by default in every setup we do, and we recommend not to modify this, unless adding additional listeners.  So provided the user sticks to our supported configuration, there shouldn't be a need for this change.

Reply:
So up until this PR we have components that talk to kafka over their designated listener, and there is a var naming which listener to use: https://github.com/confluentinc/cp-ansible/blob/5.5.x/roles/confluent.variables_handlers/defaults/main.yml#L257

One would expect that all parts of ksql would connect to kafka over the same listener, but alas that does not work in the case of the processing log WHEN rbac is enabled. My point is in a non rbac installation I would expect the interbroker listener to only be used for interbroker connections.

My suggestion is to create a new variable under line 257 like so:
```
ksql_processing_log_kafka_listener_name: "{{kafka_broker_inter_broker_listener_name if rbac_enabled else ksql_kafka_listener_name}}"
```

And then use that variable everywhere in this pr instead of `kafka_broker_inter_broker_listener_name`

Reply:
Really good point.  I am wondering though, is it more or less confusing for this feature to always connect in the same way (current implementation), or to switch depending on the auth mechanism (your suggestion).  What makes more sense to the end user?

Reply:
To confirm our conversation, using a variable would the user the opportunity to override the listener used for this feature if they do not want to use the inter-broker listener for security reasons.

If we replace inter-broker listener with ksql_processing_log_kafka_listener_name: variable should give the flexibility.